data/moves/descriptions.asm

```diff
@@ -260,8 +261,114 @@ MoveDescriptionsGen2:
 	dw RockSmashDescription
 	dw WhirlpoolDescription
 	dw BeatUpDescription
+	.IndirectEnd::
```

Comment:
```suggestion
.IndirectEnd::
```

Reply:
Oh that's just gross. Completely overlooked this.